ashwin/Structure/structure-styles.css

```diff
@@ -0,0 +1,39 @@
+body {
+    text-align: center
+}
+
+body > section {
+    width: 28%;
+    display: inline-block;
+    border: 2px solid #808080;
+    margin: 1%;
+    height: 68vh;
+    background-color: #d3d3d3
+}
+
+header {
+    border: 2px solid #808080;
+    margin-top: 3vh;
+    margin-left: 1vw;
+    margin-right: 1vw;
+    background-color: #d3d3d3
+}
+
+footer {
+    border: 2px solid #808080;
+    margin-top: 1vh;
+    margin-left: 1vw;
+    margin-right: 1vw;
+    margin-bottom: 2vw;
+    background-color: #d3d3d3
+}
+
+p {
```

Comment:
Why font styles specific to p tag?

Reply:
I did that, because I need to change the font style(to Arial) of the text inside the <p> tag and also I don't have text anywhere else apart from the <p> tag :)